docs/actions.md

```diff
@@ -49,19 +49,19 @@ Follow these steps to set up JIRA integration:
           or
         * Token: your Personal Access Tokens.
 
-Key | Description | Possible Values
---- | --- | ---
-*url* | Jira project url |
-*project-key* | The JIRA project key |
-*user* | Jira user. Use email for Jira Cloud and UserName for Jira Server/Data Center |
-*password* | Optional: User's password. API token can also be used for Cloud Jira instances. |
-*token* | Optional: User's Personal Access Token. Used only for Jira Server/Data Center |
-*board* |  Optional: JIRA board key |
-*priority*|  Optional: ticket priority, e.g., High |
-*assignee*| Optional: comma separated list of users (emails) that will be assigned to ticket, e.g., ["john@yahoo.com"]. To assign a ticket to the Application Owner email address (as defined in Aqua Application Scope, owner email field), specify ["<%application_scope_owner%>"] as the assignee value |
-*issuetype*| Optional: issue type, e.g., Bug |
-*labels*| Optional: comma separated list of labels that will be assigned to ticket, e.g., ["label1", "label2"]|
-*sprint*| Optional: Sprint name, e.g., "3.5 Sprint 8" |
+|Key           | Description          | Possible Values | Required |
+|--------------|----------------------|-----------------|----------|
+|*url*         | Jira project url     |                 | YES      |
+|*project-key* | The JIRA project key |                 | YES      |
+|*user*        | Jira user. Use email for Jira Cloud and UserName for Jira Server/Data Center    |             | YES         |
+|*password*    | Optional: User's password. API token can also be used for Cloud Jira instances. |             | NO          |    
+|*token*       | Optional: User's Personal Access Token. Used only for Jira Server/Data Center   |             | NO          |
+|*board*       |  Optional: JIRA board key              |           | NO          |
+|*priority*    |  Optional: ticket priority, e.g., High |           | NO          |
+|*assignee*    | Optional: comma separated list of users (emails) that will be assigned to ticket, e.g., ["john@yahoo.com"]. To assign a ticket to the Application Owner email address (as defined in Aqua Application Scope, owner email field), specify ["<%application_scope_owner%>"] as the assignee value   |               | NO          | 
+|*issuetype*   | Optional: issue type, e.g., Bug        |           | NO          | 
+|*labels*      | Optional: comma separated list of labels that will be assigned to ticket, e.g., ["label1", "label2"]|         | NO          |  
+|*sprint*      | Optional: Sprint name, e.g., "3.5 Sprint 8" |      | NO          |
```

Comment:
As mentioned, can we remove the "optional" and "required" strings from the description column?